tests/release_tests/test_launch/test_kubernetes.py

```diff
@@ -126,3 +88,50 @@ def test_kubernetes_agent_in_cluster():
     finally:
         # Cleanup
         cleanup_deployment(NAMESPACE)
+
+
+@pytest.mark.timeout(360)
+def test_kubernetes_agent_in_cluster_sweep():
+    init_agent_in_launch_cluster(NAMESPACE)
+    run_cap = 4
+    try:
+        sweep_config = {
+            "job": f"{ENTITY}/{PROJECT}/{JOB_NAME}",
+            "project": PROJECT,
+            "entity": ENTITY,
+            "run_cap": run_cap,
+            "method": "bayes",
+            "metric": {
+                "name": "avg",
+                "goal": "maximize",
+            },
+            "parameters": {
+                "param1": {
+                    "min": 0,
+                    "max": 8,
+                }
+            },
+        }
+
+        yaml.dump(sweep_config, open("c.yml", "w"))
+
+        proc = run_cmd_async(f"wandb launch-sweep c.yml -q {QUEUE}")
```

Comment:
We likely want `proc.kill()` in the `finally` block below to keep this self-contained